python/cudf/cudf/_lib/orc.pyx

```diff
@@ -75,10 +217,29 @@ cpdef read_raw_orc_statistics(filepath_or_buffer):
     if isinstance(filepath_or_buffer, NativeFile):
         filepath_or_buffer = NativeFileDatasource(filepath_or_buffer)
 
-    cdef raw_orc_statistics raw = (
-        libcudf_read_raw_orc_statistics(make_source_info([filepath_or_buffer]))
+    cdef parsed_orc_statistics parsed = (
+        libcudf_read_parsed_orc_statistics(make_source_info([filepath_or_buffer]))
     )
-    return (raw.column_names, raw.file_stats, raw.stripes_stats)
+
+    cdef vector[column_statistics] file_stats = parsed.file_stats
+    cdef vector[vector[column_statistics]] stripes_stats = parsed.stripes_stats
+    parsed_file_stats = []
+    parsed_stripes_stats = []
+
+    for column_index in range(file_stats.size()):
+        parsed_file_stats.append(
+            _parse_column_type_statistics(file_stats[column_index])
+        )
+
+    for stripe_index in range(stripes_stats.size()):
+        stripe_stats = []
+        for column_index in range(stripes_stats[stripe_index].size()):
+            stripe_stats.append(
+                _parse_column_type_statistics(stripes_stats[stripe_index][column_index])
+            )
+        parsed_stripes_stats.append(stripe_stats)
```

Comment:
Could do a nested list comprehension, but if that is unwieldy maybe just for the inner (`stripe_stats`) then append to `parsed_stripes_stats`. I trust your judgment on what looks best.